layouts/index.html

```diff
@@ -6,6 +6,13 @@
 {{ $kapacitorVersion := replaceRE "v" "" .Site.Data.products.kapacitor.latest }}
 {{ $fluxVersion := replaceRE "v" "" .Site.Data.products.flux.latest }}
 
+<!--
+  Show the page if:
+  - This is a regular page (not test-only) OR
+  - This is a test-only page BUT we're currently in the testing environment
+-->
+{{ if or (not .Params.test_only) (and .Params.test_only (eq site.Params.environment "testing")) }}
```

Comment:
I would add "development" here too. I use this page to add/update styles and shortcodes on a single page.

```suggestion
{{ if or (not .Params.test_only) (and .Params.test_only (in site.Params.environment (slice "testing" "development")) }}
```